pkg/distributor/distributor_test.go

```diff
@@ -1553,10 +1554,6 @@ func TestDistributor_Push_HistogramValidation(t *testing.T) {
 				assert.Nil(t, resp)
 				checkGRPCError(t, tc.expectedErr, expectedDetails, err)
 			}
-
-			t.Cleanup(func() {
```

Comment:
Note to reviewers: this is already done by `prepare()`. I removed it both here and in few other tests.